dhis-2/dhis-e2e-test/src/main/java/org/hisp/dhis/actions/metadata/ProgramActions.java

```diff
@@ -59,6 +59,9 @@ public ApiResponse createProgram( String programType )
     {
         JsonObject object = getDummy( programType );
 
+        if (programType.equalsIgnoreCase( "with_registration" )) {
```

Comment:
Codestyle, space after ( and before ).

We should probably apply the speedy spotless plugin to the e2e project as well.

Reply:
Thanks, fixed! Good point about speedy spotless  👍